### ISSUE TITLE
Launching the Lexinomicon

### ISSUE THREAD

**Issue by Flaxscrip** [cryptotechguru/Lexinomicon#18]
Let's discuss what steps are missing before we launch the Lexinomicon to the public. I believe it can be launched once issues [15](https://github.com/ulex-opensource/Lexinomicon/issues/15) and [17](https://github.com/ulex-opensource/Lexinomicon/issues/17) are resolved.

We also need to decide *how* we want to announce the Lexinomicon. I suggest we make an announcement on ulex.app. Maybe we tie it to @macterra's earlier contribution https://ulex.app/post/5cbb8f47516793520039d8de. 

It would be helpful if we can get Tom's public endorsement once we feel ready to ask for it. The Lexinomicon will be a real-world application of double democracy. There is a lot of potential packaged in there. Let's use this issue thread to play/plan our launch.

**Comment by joestartupsocieties** [comment]
I will make a post in the Ulex app. I will use the social media sharing tools to share to all startup society/icg related FB groups/pages. Afterwards, I will share the post in groups related to DLT, competitive governance, and politics. We'll also share to the startup societies and ulex telegram groups. Lastly, I'll share to the Startup Societies twitter. 

Tom is away for the rest of the week, but I can talk to him early next week about promoting it. Perhaps it would also make sense to do a podcast about the topic.

**Comment by Flaxscrip** [comment]
One part of the process we never fully developed in the Cryptonomicon is the role & content of the Bounty page in BTCPay. I believe this role should be left to a minimum but it should provide enough context to a person landing on the crowdfunding page before visiting the Lexinomicon.

The current "demo" page is here: https://bounty.cryptotech.guru/apps/355GiWY7juvhxAmxByXrVZkDDmDm/crowdfund

**Comment by Flaxscrip** [comment]
![image](https://user-images.githubusercontent.com/7094976/59527737-6618da00-8eaa-11e9-91de-2f72ef5634b3.png)

**Comment by Flaxscrip** [comment]
That is the current formal definition of a Bounty according to the Lexinomicon and the minimal information it requires.

**Comment by Flaxscrip** [comment]
![image](https://user-images.githubusercontent.com/7094976/59527822-a4ae9480-8eaa-11e9-8fae-2c4a02b77481.png)

**Comment by Flaxscrip** [comment]
The last image shows the information that is included in the current demo bounty page on BTC Pay. 
Mandatory information includes: 
1- Title
2- Funraiser Status

Optional information (to be discussed and agreed upon here):
3- Bounty Description
4- Bounty Details
5- Media
6- Contribution Levels
7- Contribution Rewards

**Comment by macterra** [comment]
As for 3, 4, 5 the approach we took with Cryptonomicon was to keep most the information in the wiki and link to the wiki page from the bounty description. Probably best to keep the info in one place to minimize redundancy and possible inconsistencies.

**Comment by Flaxscrip** [comment]
Agreed - We should keep the content in BTC Pay as static as possible with few variable parts to avoid duplication. We just didn't design something that is very presentable at the moment. I'd recommend we add the Ulex logo with a standard title and links to a Lexinomicon Bounty, link to Ulex.app for context, disclaimers, legal stuff,
